omod/pom.xml

```diff
@@ -113,6 +113,26 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>referenceapplication-omod</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>kenyaemr-api</artifactId>
+            <scope>compile</scope>

```

Comment:
please revise compile to provided

Reply:
those might have been fixed in the sub sequent commits

Reply:
okay

Reply:
Done